index.html

```diff
@@ -64,5 +66,182 @@ <h1>I’m MERCY <br />Glad to see <br />you!</h1>
         </ul>
       </div>
     </section>
+    <!--Start of detailed section-->
+    <section class="projects-section" >
+      <!--Card1-->
+      <div class="cards">
+        <img src="/assets/images/project1.png" alt="project1" />
+      </div>
+      <div class="cards-details">
+        <div>
+        <h2 class="title">Tonic</h2>
+        <div class="highlights">
+          <p class="highlight-text">Canopy</p>
+            <img class="counter" src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">Back End Dev</p>
+            <img src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">2015</p>
+        </div>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
```

Comment:
Well done here 💯  but to improve it even more the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them should be a `li`. This will help browsers understand your page more. 😉